tests/test_api.py

```diff
@@ -149,3 +150,24 @@ def test_chart_line(self, grep):
         self.assertIn(
             b'<svg xmlns:xlink="http://www.w3.org/1999/xlink', resp.get_data()
         )
+
+    def test_healthz_liveness(self):
+        """Test the /healthz/live check endpoint"""
+        result = self.client.get("/healthz/live")
+        assert result.status_code == 200
+        assert result.data == b"OK\n"
+
+    def test_healthz_readiness_ok(self):
+        """Test the /healthz/ready check endpoint"""
+        with mock.patch("datagrepper.app.readiness"):
```

Comment:
We should not mock the code that we're testing, otherwise we end up testing the mock and not the real function. In this case, `datagrepper.app.readiness` is the function that this testcase is testing, no?